### DISCUSSION TITLE
UNIX port : make USER_C_MODULES = ../../examples/usercmodule ==> not included

### DISCUSSION THREAD

**Discussion by picpic020960** [micropython/micropython#12851] [Unix & Windows] [answered]
Bonjour ,
with the doc   `https://docs.micropython.org/en/latest/develop/cmodules.html` 
I tests 
 **Compiler le module c dans MicroPython**
but no cexample module included.
Sure , some trikks
Please help.
NB : OK for ESP32 port with other example but KO for unix port. Maybe garbaage from make/cmake conversion

**Comment by picpic020960** [comment, marked as answer]
OK with VARIANT=coverage in make parameter

**Reply by jimmo** [reply]
Maybe if you had previously build the (standard variant of the) unix port, you need to do `make clean` first.
